crates/fuel-core/src/schema/gas_price.rs

```diff
@@ -0,0 +1,51 @@
+use super::scalars::U64;
+use crate::{
+    fuel_core_graphql_api::{
+        database::ReadView,
+        Config as GraphQLConfig,
+    },
+    query::BlockQueryData,
+};
+use async_graphql::{
+    Context,
+    Object,
+};
+use fuel_core_types::blockchain::block::Block;
+
+pub struct LatestGasPrice {
+    pub gas_price: U64,
+    pub block_height: U64,
```

Comment:
Nit: This could probably be a `U32` since block height is based on a `u32`

Reply:
Thanks. Nice catch. I meant to switch these earlier and I forgot. I can't remember why I didn't do `U32` to start out.